AirSkyBoat/AirSkyBoat
# Update Bugbear Mask desynth for leatherwork to accurate results

**_I affirm:_**

## What does this pull request do?



Updates the values for the Leatherwork desynth of Bugbear Masks to properly reflect the current (and furture) results for HQ synths.  https://ffxiclopedia.fandom.com/wiki/Bugbear_Mask?oldid=40346

Current HQ results yield Gold Thread, and on Horizon it appears they've been updated to produce Ram Leather for HQ1 and HQ2, but in the incorrect amounts as per the bug report [here](https://discord.com/channels/933423693848260678/1062159694547910716/1062159694547910716)


## Steps to test these changes

1. Update synth_recipes table 
2. Obtain a Bugbear Mask 
3. Set Leatherwork to 86+ for best results (T3)
4. Desynth with Wind Crystal. All HQ's should yield 3 Ram Leather, except HQ3 which yields 2 Gold Threads
